book/src/chap01.asciidoc

```diff
@@ -148,172 +148,172 @@ julia> typeof("42.0")
 
-This is @ well-structured Engli$h sentence with invalid t*kens in it. This sentence all valid tokens has, but invalid structure with.
+Essa é um@ sentença bem estruturada em portuguê$ com t*kens inválidos. Essa frase possui todos os tokens válidos, mas com estrutura invalida.
 
-When ((("parsing")))you read a sentence in English or a statement in a formal language, you have to figure out the structure (although in a natural language you do this subconsciously). This process is called _parsing_.
+Quando ((("análise"))) você lê uma sentença em Poruguês ou uma declaração numa linguagem formal, você tem que descrever a estrutura (apesar de que em uma linguagem natural você faz isso subconscientemente). Esse processo é chaamdo de _análise_.
 
-Although formal and natural languages have many features in common—tokens, structure, and syntax—there are some differences:
+Embora as linguagens formais e naturais tenham muito recursos em comuns — tokens, estruturas, e sintaxes —  existem algumas diferenças:
 
-Ambiguity::
-Natural languages are full of ambiguity, which people deal with by using contextual clues and other information. Formal languages are designed to be nearly or completely unambiguous, which means that any statement has exactly one meaning, regardless of context.
+Ambiguidade::
+Linguagens naturais são cheias de ambiguidades, com pessoas acordando com elas e usando em pistas contextuais e outras informações. Linguagens formais são designadas para serem próximas e completamente desambígua sem ambiguidade, o que significa que toda declaração possui exatamente um significado, independentemente do contexto.
 
-Redundancy::
-In order to make up for ambiguity and reduce misunderstandings, natural languages employ lots of redundancy. As a result, they are often verbose. Formal languages are less redundant and more concise.
+Redundância::
+Para compensar a ambiguidade e reduzir os mal-entendidos, as línguas naturais empregam muita redundância. Como resultado, eles geralmente são detalhados. Linguagens formais são menos redundantes e mais concisas.
 
-Literalness::
-Natural languages are full of idiom and metaphor. If I say, “The penny dropped,” there is probably no penny and nothing dropping (this idiom means that someone understood something after a period of confusion). Formal languages mean exactly what they say.
+Literalidade::
+As línguas naturais estão cheias de expressões idiomáticas e metáforas. Se eu disser: "O centavo caiu", provavelmente não há centavo e nada está caindo (nesse idioma significa que alguém entendeu alguma coisa após um período de confusão). Linguagens formais significam exatamente o que dizem.
```

Comment:
The penny dropped é uma expressão, nas faz sentido a tradução literal.
Nosso equivalente é "a ficha caiu".